astro_2.0/features/CreateProposal/components/CreateProposalCard/CreateProposalCard.tsx

```diff
@@ -343,8 +349,8 @@ export const CreateProposalCard: React.FC<CreateProposalCardProps> = ({
           titlePlaceholder="Add draft name"
           title={title}
           setTitle={handlerChangeTitle}
-          hashtags={hashtags}
-          setHashtags={handlerChangeHashtags}
+          // hashtags={hashtags}
+          // setHashtags={handlerChangeHashtags}
```

Comment:
Do we need this props?

Reply:
Hashtags in drafts will be removed next PR